central/graphql/resolvers/clusters.go

```diff
@@ -54,9 +55,9 @@ func init() {
 			"imageCount(query: String): Int!",
 			"components(query: String, pagination: Pagination): [EmbeddedImageScanComponent!]!",
 			"componentCount(query: String): Int!",
```

Comment:
Nit/Minor : `componentCount` would be before `components` in alphabetical order. Would apply for most fields except fora few like vulnerabilities.

However, it doesn't have to be a strict alphabetical order and it looks more consistent this way.